docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,246 @@
+# Institutional FAQ
+
+This page contains common questions from users of JupyterHub,
+broken down by their roles within organizations.
+
+# For all
+
+## Is appropriate for adoption within a larger institutional context?
+
+Yes! JupyterHub has been used at-scale for large pools of users, as well
+as complex and high-performance computing. For example, UC Berkeley uses
+JupyterHub for its Data Science Education Program courses (serving over
+3,000 students). The Pangeo project uses JupyterHub to provide access
+to scalable cloud computing with Dask. JupyterHub is stable customizable
+to the use-cases of large organizations.
+
+## I keep hearing about Jupyter Notebook, JupyterLab, and now JupyterHub. What’s the difference?
+
+Here is a quick breakdown of these three tools:
+
+* **The Jupyter Notebook** is a document specification (the `.ipynb`) file that interweaves
```

Comment:
Maybe add that there are many tools that read and write "Jupyter notebooks the file format": nteract, jupyter, google colab, kaggle kernels, etc

Or do you think it will cause more confusion than help?